main.tf

```diff
@@ -0,0 +1,215 @@
+module "label" {
```

Comment:
Probably need a more specific name than `label`, since there will often be more than one label per module.

Reply:
`main.tf` needs `terraform fmt`